### PR TITLE
Fix #2270 Added RTL mode for Audio and Language Activity

### LINKED ISSUE #2270: [RTL] High-fi Audio/App Language
`Audio/App Language` needs RTL support as shown in below screen.
OP (Default Audio Language) – RTL

Make sure that this works correctly in mobile (landscape + portrait) and tablet (landscape + portrait).

## References
Mobile Portrait: https://xd.adobe.com/view/e8aa4198-3940-47f9-514a-f41cc54457f6-9e9b/grid/
Mobile Landscape: https://xd.adobe.com/view/ee9e607b-dbd6-4372-48dc-b687d32af3da-98af/grid/
Tablet: https://xd.adobe.com/view/d405de00-a871-4f0f-73a0-f8acef30349b-a234/grid/
Wiki: https://github.com/oppia/oppia-android/wiki/Working-on-UI

### PR DESCRIPTION
**Pull request by imraghav20**
Fixes #2270

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

File: .idea/codeStyles/Project.xml [hunk 1]
```diff
@@ -5,6 +5,22 @@
...
+      </option>
+      <option name="PACKAGES_IMPORT_LAYOUT">
+        <value>
+          <package name="" alias="false" withSubpackages="true" />
+          <package name="java" alias="false" withSubpackages="true" />
+          <package name="javax" alias="false" withSubpackages="true" />
+          <package name="kotlin" alias="false" withSubpackages="true" />
+          <package name="" alias="true" withSubpackages="true" />
+        </value>
+      </option>
```
Comment (FareesHussain) [inline .idea/codeStyles/Project.xml:23]: Please don't make changes in `.idea` files.
Have a look at the checklist

Comment (FareesHussain) [inline .idea/codeStyles/Project.xml]: Please don't make changes in `.idea` files.
Have a look at the checklist

**Review by FareesHussain — CHANGES_REQUESTED** [review]
@imraghav20 Add screenshots in the description
Have a look at RTL guidelines for the screenshots
https://github.com/oppia/oppia-android/wiki/RTL-Guidelines

**Comment by imraghav20** [comment]
> @imraghav20 Add screenshots in the description
> Have a look at RTL guidelines for the screenshots
> https://github.com/oppia/oppia-android/wiki/RTL-Guidelines

Added screenshots in both portrait and landscape mode.

WhatsApp Image 2020-12-29 at 4 16 29 PM (1)
WhatsApp Image 2020-12-29 at 4 16 29 PM (2)
WhatsApp Image 2020-12-29 at 4 16 29 PM (3)
WhatsApp Image 2020-12-29 at 4 16 29 PM

**Comment by FareesHussain** [comment]
@imraghav20 please revert the changes in  `Project.xml`

**Comment by imraghav20** [comment]
> @imraghav20 please revert the changes in `Project.xml`

Yes I am trying to do so, but whenever after changes I check out `git status` it shows there are no changes to commit. And when I go back to Android Studio, I see the code has been auto generated.

Any suggestions?

**Comment by FareesHussain** [comment]
> > @imraghav20 please revert the changes in `Project.xml`
> 
> Yes I am trying to do so, but whenever after changes I check out `git status` it shows there are no changes to commit. And when I go back to Android Studio, I see the code has been auto generated.
> 
> Any suggestions?

@imraghav20 You can change it directly from github and paste the code from the following file
https://github.com/oppia/oppia-android/blob/develop/.idea/codeStyles/Project.xml

**Comment by imraghav20** [comment]
> > > @imraghav20 please revert the changes in `Project.xml`
> > 
> > 
> > Yes I am trying to do so, but whenever after changes I check out `git status` it shows there are no changes to commit. And when I go back to Android Studio, I see the code has been auto generated.
> > Any suggestions?
> 
> @imraghav20 You can change it directly from github and paste the code from the following file
> https://github.com/oppia/oppia-android/blob/develop/.idea/codeStyles/Project.xml

Thanks @FareesHussain

**Review by FareesHussain — APPROVED** [review]
LGTM, thanks

**Review by rt4914 — APPROVED** [review]
LGTM, thanks.

**Pull request merged**